mxnet/inference/docker/artifacts/config.properties

```diff
@@ -1,4 +1,4 @@
-vmargs=-XX:-UseContainerSupport
+vmargs=-XX:+UseContainerSupport -XX:InitialRAMPercentage=8.0 -XX:MaxRAMPercentage=10.0 -XX:-UseLargePages -XX:+UseG1GC -XX:+ExitOnOutOfMemoryError
```

Comment:
Is this change still needed ?

Reply:
Sorry. I was using this branch to test. Will remove it